### PR TITLE
Iframe API 

### PR DESCRIPTION
**Pull request by Gusted**

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -14,26 +23,31 @@ const isIFrame = (() => {
...
         throw new Error('Theme engine is not supported');
     }
 
+
     createOrUpdateDynamicTheme(theme, fixes, isIFrame);
+    const frames = document.getElementsByTagName("iframe");
+    const script = document.createElement('script');
+    script.src = 'https://cdn.jsdelivr.net/npm/darkreader@latest/darkreader.min.js'
+    const scriptEnable = document.createElement('script');
+    scriptEnable.textContent = `DarkReader.enable()`;
```
Comment (alexanderby) [inline src/api/index.ts:39]: FYI, the script will be executed before the remote script. There are several options like `defer` etc.

File: tasks/bundle-api.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ async function bundleAPI() {
                 tsconfigOverride: {
                     compilerOptions: {
                         removeComments: true,
-                        target: 'es5',
+                        target: 'ES6',
```
Comment (alexanderby) [inline tasks/bundle-api.js:24]: This change is not related to iframes. I didn't analyze how the browser market changed in the meantime.

**Comment by Gusted** [comment]
Understand that

File: tasks/bundle-api.js (hunk 2 above)
Comment (Gusted) [reply tasks/bundle-api.js:24]: well.. your new array.ts use ES6 :)

File: src/api/index.ts [hunk 3]
```diff
@@ -14,26 +23,31 @@ const isIFrame = (() => {
...
 
     if (theme.engine !== ThemeEngines.dynamicTheme) {
         throw new Error('Theme engine is not supported');
     }
 
+
     createOrUpdateDynamicTheme(theme, fixes, isIFrame);
+    const frames = document.getElementsByTagName("iframe");
+    const script = document.createElement('script');
+    script.src = 'https://cdn.jsdelivr.net/npm/darkreader@latest/darkreader.min.js'
```
[the remark this answers is not in the document]
Comment (Gusted) [reply src/api/index.ts:37]: We can't export an module trough contenWindow

File: src/api/index.ts [hunk 4]
```diff
@@ -5,6 +5,15 @@ import {Theme, DynamicThemeFix} from '../definitions';
 import ThemeEngines from '../generators/theme-engines';
 import {createOrUpdateDynamicTheme, removeDynamicTheme} from '../inject/dynamic-theme';
 
+
+export const setFetchMethod = setFetch;
```
[the remark this answers is not in the document]
Comment (Gusted) [reply src/api/index.ts:9]: VsCode issues with saying `undetectable code detected`
